addons/l10n_lt/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,548 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="account_account_tag_1" model="account.account.tag">
+    <field name="name">A. FIXED ASSETS</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+  <record id="account_account_tag_2" model="account.account.tag">
+    <field name="name">A.1. INTANGIBLE ASSETS</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
```

Comment:
I don't think you need the upper levels here, only the lowest.  You can add them in the financial report as the sum of the lower lines.

Reply:
You mean move suchs tags in report module if needed (that would be added in enterprise repository)?

Reply:
You just do not need the tags.  You can tell a financial report line to be the sum of other lines.

Reply:
OK, I will recheck and remove not used tags and keep only those that are needed for accounts directly.